cards/angioedema/card.md

```diff
@@ -4,7 +4,7 @@ title: Angioedema
 authors:
     - 'Michelle Lin, MD'
 created: 2010/03/26
-updates: null
+updates: 2017/2/24
 categories:
     - Allergy-Immunology
 drugs: 'ACE inhibitor'
```

Comment:
after internal discussion with the dev team, we're dropping the drugs, so these can be deleted. I will do a bulk edit on all cards so don't worry about making the change, just FYI